README.md

```diff
@@ -44,6 +45,14 @@ This has been tested and works on:
 
 - CentOS 8
 
+## Security
+
+The default configuration enables direct connections to the VNC ports through iptables rules. If you are running the host for VNC and crc on an Internet-connected system, this is not a good idea. Instead, update the [vars/main.yml](./vars/main.yml) file changing `allowDirect` to false and instead tunnel your VNC connections through ssh using something like:
+
+```console
+ssh -L5901:127.0.0.1:5901 youruser@yourhost
```

Comment:
@jjasghar this was my stab at the tips on how to encapsulate the connection... is it good enough?

Reply:
Wunderbar!